dmoj/executors/base_executor.py

```diff
@@ -54,6 +54,8 @@
         ExactFile('/dev/hv_tsc'),
         RecursiveDir('/dev/fd'),
     ]
+
+    BASE_WRITE_FILESYSTEM += [ExactFile('/dev/stderr')]
```

Comment:
Worth adding a comment: https://github.com/DMOJ/judge-server/commit/08791138854cdb5c1f131e7f3d496cdff1ccb80b

Reply:
`/dev/stderr` actually fixes nothing on PASCAL: I believe the reason we have sporadic TLEs now is because of the cost of outputting to the screen, which wasn't present before, because we had this logging on DEBUG.

Reply:
That's hilarious.